cranelift-codegen/src/isa/x86/abi.rs

```diff
@@ -509,17 +556,21 @@ fn fastcall_prologue_epilogue(func: &mut ir::Function, isa: &dyn TargetIsa) -> C
         panic!("TODO: windows-fastcall: x86-32 not implemented yet");
     }
 
-    let csrs = callee_saved_gprs_used(isa, func);
+    let csrs = callee_saved_regs_used(isa, func);
 
     // The reserved stack area is composed of:
     //   return address + frame pointer + all callee-saved registers + shadow space
     //
+    // XMM registers are twice the size of general purpose registers, so count them
+    // twice for stack space accounting.
```

Comment:
Is this comment out of date? It's using `types::F64X2.bytes() as usize` now which doesn't look like double-counting!

Reply:
you are correct! I realized shortly after writing the comment that double-counting is wrong for 32-bit targets, but forgot to fix the comment too.